src/utils/validators.js

```diff
@@ -24,6 +25,8 @@ const RESOURCE_CATEGORY_REGEX = '^([a-zA-Z0-9]+[- ])*[a-zA-Z0-9]+$';
 const resourceRoomNameRegexTest = /^([a-zA-Z0-9]+-)*[a-zA-Z0-9]+$/
 const resourceCategoryRegexTest = RegExp(RESOURCE_CATEGORY_REGEX);
 
+const ISOMER_TEMPLATE_PROTECTED_DIRS = ['data', 'includes', 'site', 'layouts', 'files', 'images', 'misc', 'pages']
```

Comment:
Suggestion: might be a good idea to display this in the folder naming modal

Reply:
Good point, as discussed, we will put this in the error message for now as this should not be a common occurrence. Changed in https://github.com/isomerpages/isomercms-frontend/pull/369/commits/9a5868f843966298a92e41359e195c980eedaad8